### PR TITLE
Fix xss vulnerablities

### PR DESCRIPTION
**Pull request by sui-drmz**

### CODE REVIEW

File: includes/admin/pages/class-wp-statistics-admin-page-settings.php [hunk 1]
```diff
@@ -93,6 +93,11 @@ public function save()
                 $wp_statistics_options = self::{'save_' . $method . '_option'}($wp_statistics_options);
             }
 
+            // Sanitize Options
```
Comment (mostafasoufi) [inline includes/admin/pages/class-wp-statistics-admin-page-settings.php:96]: There is no input data in this method, all data in the method are getting from the DB.

File: includes/admin/pages/class-wp-statistics-admin-page-settings.php [hunk 2]
```diff
@@ -415,10 +420,9 @@ public static function save_general_option($wp_statistics_options)
             $disable_strip_uri_parameters = true;
         }
         foreach ($selist as $se) {
-            $se_post     = 'wps_disable_se_' . $se['tag'];
-            $optionValue = isset($_POST[$se_post]) ? sanitize_text_field($_POST[$se_post]) : '';
```
Comment (mostafasoufi) [inline includes/admin/pages/class-wp-statistics-admin-page-settings.php:419]: I wonder why the `sanitize_text_field()` has been removed.

File: includes/admin/pages/class-wp-statistics-admin-page-settings.php [hunk 3]
```diff
@@ -450,8 +454,7 @@ public static function save_general_option($wp_statistics_options)
         }
 
         foreach ($wps_option_list as $option) {
-            $optionValue                                                = isset($_POST[$option]) ? sanitize_text_field($_POST[$option]) : '';
-            $wp_statistics_options[self::input_name_to_option($option)] = $optionValue;
+            $wp_statistics_options[self::input_name_to_option($option)] = (isset($_POST[$option]) ? $_POST[$option] : '');
```
Comment (mostafasoufi) [inline includes/admin/pages/class-wp-statistics-admin-page-settings.php:457]: Shouldn't be sanitized the data here?

File: includes/admin/pages/class-wp-statistics-admin-page-settings.php (hunk 1 above)
Comment (sui-drmz) [reply includes/admin/pages/class-wp-statistics-admin-page-settings.php:96]: as i went thorough save_ methods  i realized they replace "db" values with "input" values using $_POST [$option_name]
so after t that loop( which triggers self::save_  methods ) $wp_statistics_options reflect user input if provided , if not they have a value of ' ' .

File: includes/admin/pages/class-wp-statistics-admin-page-settings.php (hunk 3 above)
Comment (sui-drmz) [reply includes/admin/pages/class-wp-statistics-admin-page-settings.php:457]: so what about other save_ methods?

File: includes/admin/pages/class-wp-statistics-admin-page-settings.php (hunk 1 above)
Comment (mostafasoufi) [reply includes/admin/pages/class-wp-statistics-admin-page-settings.php:96]: The belows should be sanitized
```
$value = $_POST['user_custom_header_ip_method']
$value = $_POST['ip_method'];
```

**Pull request closed without merging**